CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+* Remove usage of goog.isFunction.
+* Fix FirebaseUI-web spinner off for RTL languages.
+* Show warning on accountchooser.com becomes universally opt-out.
```

Comment:
* Log warning on accountchooser.com being deprecated and switching to universal opt-out mode starting in July 31st, 2020.